packages/check-iam-wildcards/checkAsterisks.ts

```diff
@@ -0,0 +1,61 @@
+import { iam } from './security.serverless';
+import { IamRole } from 'serverless/aws';
+
+const yaml = require('js-yaml');
+const fs = require('fs');
+
+const project: any = process.env.PROJECT;
```

Comment:
**chore |** there not necessity to use `any` by default it will be `string | undefined`. We can add extra verification in **main** as example, just to be sure that PROJECT was defined

```typescript
async function main() {
  if (!project) {
    throw new Error('PROJECT env variable should be defined');
  }
  console.log('Going to check asterisks in iam policy ...');
  const data: any = getIam();
  checkAsterisks(data);
}
```

Reply:
ok, I'm going to add but it checks before, project name is required input for the action.